src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -71,67 +70,61 @@ import {
 import { FetchStatus } from '../../../services/utils';
 import { copyOrMoveModifications } from '../../../services/study';
 import { MODIFICATION_TYPES } from 'components/utils/modification-type';
+import { Box } from '@mui/system';
 
-const useStyles = makeStyles((theme) => ({
-    listContainer: {
+const styles = {
+    listContainer: (theme) => ({
         overflowY: 'auto',
         display: 'flex',
         flexDirection: 'column',
         flexGrow: 1,
         paddingBottom: theme.spacing(8),
-    },
-    list: {
+    }),
+    list: (theme) => ({
         paddingTop: theme.spacing(0),
         flexGrow: 1,
-    },
-    modificationsTitle: {
+    }),
+    modificationsTitle: (theme) => ({
         display: 'flex',
         alignItems: 'center',
         margin: theme.spacing(0),
         padding: theme.spacing(1),
         backgroundColor: theme.palette.primary.main,
         color: theme.palette.primary.contrastText,
         overflow: 'hidden',
-    },
-    toolbar: {
-        padding: theme.spacing(0),
+    }),
+    toolbar: (theme) => ({
+        padding: '0 !important', // TODO CHARLY tester avec "&:" ou déclinaisons
```

Comment:
TODO ? Now or in another PR ?

And I'm not sure "&:" are working in emotion

Also, why do we need "important" now ?

Reply:
Yes sorry, this old comment is not supposed to be pushed